zookeeper-server/src/main/java/org/apache/zookeeper/server/ServerMetrics.java

```diff
@@ -455,6 +457,8 @@ private ServerMetrics(MetricsProvider metricsProvider) {
 
     public final Counter CNXN_CLOSED_WITHOUT_ZK_SERVER_RUNNING;
 
+    public final Counter SKIP_LEARNER_REQUEST_TO_NEXT_PROCESSOR_COUNT;
```

Comment:
I see that introducing this counter helps in unit testing. Other than unit testing, do you think this counter will come handy for the customers during production? 
- If yes, then we might document this (but I am not sure if we usually mention counters in the admin docs).
- If no, then do you think we would have any other way for testing this, without adding an unnecessary metrics?

Reply:
@mayank99fb will probably confirm this, but I think this was intentional. It's a useful thing to monitor the effect of such a new feature when you turn on first time in production. We should keep it.

Reply:
@symat this is used to monitor the feature, internally, we have the metrics reporting system which will read and report from this value.

Reply:
@symat , as @anmolnar and @lvfangmin mentioned - the intention of this counter was to help in monitoring and I would suggest me keeping it in the code unless you still have some concerns on this ?

Reply:
no concerns, let's keep it then :)